packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputList.tsx

```diff
@@ -82,6 +82,7 @@ export interface TransactionReviewOutputListProps {
     outputs: ReviewOutput[];
     buttonRequestsCount: number;
     isRbfAction: boolean;
+    network?: Network;
```

Comment:
leftover?

Reply:
4a5ddbbe3ce81e4f460e46c0be13c4bec6062881

Reply:
not complete, please check failing validation

Reply:
Sorry, now it should be fixed feb1f0345d4131b63d2041a745b5534fda51fde0